### PR TITLE
MPDP-456 Adding aggregate data page

### PR DESCRIPTION
**Pull request by SidG-CG**

### CODE REVIEW

File: app/views/service-start/index.njk [hunk 1]
```diff
@@ -17,6 +17,8 @@
       <li>Farming Equipment and Technology Fund</li>
     </ul>
 
+    <p>You can also <a id='viewYearlyTotals' href="/viewSchemePaymentsByYear">view yearly totals.</a></p>
+
     {{ govukInsetText({
       html: "This service shows payments made between 1 April 2021 and 31 March 2022."
```
Comment (vmuthu-kn) [inline app/views/service-start/index.njk:23]: Why is this hard coded to only 2 years? Are we not supporting up to 5 years data?

Comment (SidG-CG) [reply app/views/service-start/index.njk:23]: That hasn't changed in this branch. We have not yet done the work to update the service start page to bring it up-to-date with scalable view yet. That will be in a separate ticket.

Comment (vmuthu-kn) [reply app/views/service-start/index.njk:23]: ok

**Review by ssunkuga — APPROVED** [review]
on behalf of sid(absence holiday) i am approving this as per vel request

**Review by ssunkuga — APPROVED** [review]

**Review by ssunkuga — APPROVED** [review]

**Review by ssunkuga — APPROVED** [review]

**Review by ssunkuga — APPROVED** [review]

**Review by ssunkuga — APPROVED** [review]

**Review by ssunkuga — APPROVED** [review]

**Pull request merged**